packages/web-app-user-management/src/components/Users/UsersList.vue

```diff
@@ -132,6 +142,29 @@ export default {
       const translated = this.$gettext('Select %{ user }')
 
       return this.$gettextInterpolate(translated, { user: user.displayName }, true)
+    },
+    getUserRole(user) {
+      const userAssignmentList = this.userAssignments.find(
+        (assignment) => assignment.accountUuid === user.id
+      )
+
+      if (!userAssignmentList) {
+        return ''
+      }
+
+      const userRoleAssignment = userAssignmentList.find((assignment) => 'roleId' in assignment)
+
+      if (!userRoleAssignment) {
+        return ''
+      }
+
+      const role = this.roles.find((role) => role.id === userRoleAssignment.roleId)
+
+      if (!role) {
+        return ''
+      }
+
+      return role.displayName
```

Comment:
`return role ? role.displayName : ''`

Reply:
I would like to stick to the code pattern, see above.

Reply:
alright fine for me